src/helperFunctions/get_config_of_plugins.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+from helperFunctions.fileSystem import get_src_dir
+import configparser
+import logging
+
+
+def get_path_to_conf():
+    return str(Path(get_src_dir()) / 'config' / 'main.cfg')
+
+
+def load_plugin_conf(input_list):
+    config = configparser.ConfigParser()
+    config.read(get_path_to_conf())
+
+    threads_info = {}
+
+    for plugin in input_list:
+        number_threads = 1
+
+        if plugin in config:
```

Comment:
This can still be optimized:
```python
thread_info.update(
    {plugin: 1 if plugin not in config else config.get(plugin, 'threads', 1)}
)
```
The warning is not necessary.

Reply:
or even
```python
thread_info = {
    plugin: 1 if plugin not in config else config.get(plugin, 'threads', 1)
    for plugin in input_list
}
```